manifests/dev/vsphere-7.0u2/guestcluster/1.17/pvcsi.yaml

```diff
@@ -362,8 +405,21 @@ data:
     [FeatureStatesConfig]
     name = "csi-feature-states"
     namespace = "{{ .PVCSINamespace }}"
+    [InternalFeatureStatesConfig]
+    name = "internal-feature-states.csi.vsphere.vmware.com"
+    namespace = "{{ .PVCSINamespace }}"
 kind: ConfigMap
 metadata:
   name: pvcsi-config
   namespace: {{ .PVCSINamespace }}
 ---
+apiVersion: v1
+data:
+  "volume-extend": "true"
+  "volume-health": "true"
+  "online-volume-extend": "false"  # online-volume-extend feature is available starting from vSphere 7.0U2
```

Comment:
Older TKG versions will have this FSS set to False. Customers cannot change this. This will be important for those customers who want this feature right?

Reply:
This gives the impression that only upgrading to 7.0u2 would enable this feature

Reply:
Okay then I will remove the comment

Reply:
Removed it